modules/virt-exposing-pci-device-in-cluster-cli.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
+
+[id="virt-exposing-pci-device-in-cluster-cli_{context}"]
+= Exposing PCI host devices in the cluster using the CLI
+
+To enable the availability of PCI host devices in the cluster, add details about the PCI devices to the `HyperConverged` custom resource (CR). The PCI devices are exposed and permitted to be used in the cluster.
+
+
+.Procedure
+. Edit the `HyperConverged` CR in YAML format using the default editor by running the following command:
```

Comment:
I might s/in YAML format using the default editor/in your default editor

(mostly because we don't usually specify that it's going to be a YAML)